### DISCUSSION TITLE
Depression length

### DISCUSSION THREAD

**Discussion by Jagge** [karttapullautin/karttapullautin#171] [General]
I can remember seeing some discusion about depression_length parameter somewhere. Anyway, one more note about it. If you process in the batch mode there will be depressions going outside a tile + buffer and those will usually not get drawn as depression no mater how high value you set. The original value was set to kind of kept the size consistent, depressions on tile's edge or middle of a tile would be drawn the same way (only small enough as depressions).

**Comment by rphlo** [comment]
the parameter you talk about is named `depression_length` is the ini and default to 181 which was the arbitrary value that was hardcoded before that.
